pkg/controller/plan/kubevirt.go

```diff
@@ -574,8 +577,10 @@ func (r *KubeVirt) getPVCs(vm *plan.VMStatus) (pvcs []core.PersistentVolumeClaim
 		pvcAnn := pvc.GetAnnotations()
 		if pvcAnn[kVM] == vmLabels[kVM] && pvcAnn[kPlan] == vmLabels[kPlan] {
 			pvcs = append(pvcs, *pvc)
-		} else {
-			if r.useOvirtPopulator(vm) {
+		} else if r.isOpenstack(vm) {
```

Comment:
isn't it problematic? (we won't get to useOvirtPopulator

Reply:
yeah...

Reply:
ack, I posted this before you fixed it